R/summarize.R

```diff
@@ -1,9 +1,10 @@
 #' @export
-#' @importFrom dplyr bind_cols summarise_ select
+#' @importFrom dplyr bind_cols summarise select
 #' @importFrom purrr map_chr map_df
 #' @importFrom crunch crtabs
-summarise_.CrunchDataset <- function (.data, ..., .dots) {
-    dots <- all_dots(.dots, ..., all_named = TRUE)
+#' @importFrom lazyeval all_dots
+summarise.CrunchDataset <- function (.data, ...) {
+    dots <- lazyeval::lazy_dots(...)
```

Comment:
Looks like you're importing `all_dots` but using `lazy_dots`. If you fix your import, you can drop the `::`.